packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Relations/Relations.tsx

```diff
@@ -601,25 +590,137 @@ const RelationsInput = ({
     params: currentDocumentMeta.params,
   } as DocumentMeta;
 
+  const {
+    permissions = [],
+    isLoading: isLoadingPermissions,
+    error,
+  } = useRBAC(
+    PERMISSIONS.map((action) => ({
+      action,
+      subject: relation.model,
+    }))
+  );
+
+  if (isLoadingPermissions) {
+    return (
+      <Loader small>
+        {formatMessage({
+          id: 'content-manager.ListViewTable.relation-loading',
+          defaultMessage: 'Relations are loading',
+        })}
+      </Loader>
+    );
+  }
```

Comment:
do you think you could move the loading state inside the combobox? Otherwise I would assume this creates a layout shift when permissions load